buildSrc/src/test/java/com/osacky/flank/gradle/integration/TestFixtures.kt

```diff
@@ -1,8 +1,27 @@
 package com.osacky.flank.gradle.integration
 
+import org.gradle.testkit.runner.GradleRunner
 import org.junit.rules.TemporaryFolder
 import java.io.File
 
 fun TemporaryFolder.setupFixture(fixtureName: String) {
   File(this::class.java.classLoader.getResource(fixtureName)!!.file).copyRecursively(newFile(fixtureName), true)
 }
+
+internal fun makeGradleFile(where: TemporaryFolder, stringProvider: () -> String) = where
```

Comment:
I think this would be more straightforward if the last argument were simply a string instead of a stringProvider.
```suggestion
internal fun writeBuildDotGradle(where: TemporaryFolder, buildScript: String) = where
```